doc/docs/tutorials/theming_tutorial.md

```diff
@@ -1 +1,22 @@
 # Customising the design of your application
+
+Tutorial Outline (WIP)
+
+* Built in settings
+* Adding our own CSS file
+* Key overrides and a .sass template to reset key colours
+* Changing the layout of the application
+* Bundling as a standalone theme
+
+## Changing logos
+
+We can change the logo and favicon of our application by changing settings.
+Create a logo and a favicon, place them in `./yourapp/static/img/` and then
+update the following settings:
+
+```python
+# ./yourapp/settings.py
+
+OPAL_FAVICON_PATH = 'img/my.favicon.ico'
```

Comment:
if the path in which to place favicons is `./yourapp/static/img/`, then if feels redundant to respecify this in OPAL_FAVICON_PATH = **'img**/my.favicon.ico'.